docs/developers/strat-lib/background/offer-maker/forwarder.md

```diff
@@ -3,24 +3,26 @@ sidebar_position: 3
 ---
 # Forwarder
 
-Forwarder is an abstract implementation of [MangroveOffer](mangrove-offer.md), if you don't have a good understanding of MangroveOffer we recommend reading that page first. This page is going to compare the [Direct](direct.md) implementation of MangroveOffer with Forwarder, we recommend reading about [Direct](direct.md) before reading this page.
+[Forwarder](https://github.com/mangrovedao/mangrove-strats/blob/feat/smartRouter/src/strategies/offer_forwarder/abstract/Forwarder.sol) is an abstract implementation of [MangroveOffer](mangrove-offer.md), if you don't have a good understanding of MangroveOffer we recommend reading that page first. This page is going to compare the [Direct](direct.md) implementation of MangroveOffer with Forwarder, we recommend reading about [Direct](direct.md) before reading this page.
 
-Forwarder should be seen as an implementation the can be used by multiple offer makers. This means that anyone can manage offers using the contract. Because of this, Forwarder needs to keep track of who owns which offer and what the reserve is for the caller. This is the key difference between Direct and Forwarder.
+Forwarder should be seen as an implementation the can be used by multiple offer makers. This means that anyone can manage offers using the contract. Because of this, Forwarder needs to keep track of who owns which offer and what the reserve is for the caller. It does so by using a [`RouterProxyFactory`](https://github.com/mangrovedao/mangrove-strats/blob/feat/smartRouter/src/strategies/routers/RouterProxyFactory.sol), which creates a dedicated proxy router for each user of the Forwarder. This is the key difference between Direct and Forwarder.
 
-Forwarder does many of the same things as MangroveOffer and Direct with a few key differences.
+Useful things to note:
 
-**How inbound tokens are handled (from the taker):** After the funds have been transferred from the taker to the Forwarder contract, it transfers the funds to the reserve of the offer owner. The reason for this, is that it cannot leave the funds on the contract, since the inbound tokens need to be distributed to each offer owners. When the offer has been successfully taken, it has no funds on the contract itself (vs. Direct where the funds are still on the contract). It therefore has no extra actions and just uses the default implementation of MangroveOffer.
+## Using `RouterProxyFactory` to create `RouterProxy` instances
```

Comment:
The phrasing/formatting of this looks a bit weird:
<img width="744" alt="billede" src="https://github.com/mangrovedao/mangrove-docs/assets/3458856/25813edf-c67f-4e4c-a3e1-1e5cad57ab13">

Maybe rephrase the sentence before the sections to something like:

"The following sections explain some key aspects of Forwarder."